Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:latest
```

Comment:
Specify exact version instead of latest in all Dockerfiles

Reply:
I'll fix it in next commit

Reply:
fixed version in 20b0f04008286f5de5d78942cb282d962388b6c2